Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 	rm -f ./build.log
 
 .PHONY: build
-build:	update_submodules
+build:	update_submodules lint
```

Comment:
should this be on a new line?

Reply:
doesn't seem to work on a new line

Reply:
prerequisites have to be on same line https://makefiletutorial.com/